Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM ubuntu:bionic-20220427 as builder
+
+WORKDIR /work
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+	    build-essential \
+	    pkg-config \
+	    checkinstall \
+	    git \
+	    autoconf \
+	    automake \
+	    libtool-bin \
+	    libplist-dev \
+        libavahi-glib-dev libavahi-client-dev \
+        libimobiledevice-dev \
```

Comment:
Is this dependency necessary? rusty_libimobiledevice is being built in vendor mode which should build the source itself

Reply:
Specifically the libimobiledevice-dev

Reply:
It seems that `libimobiledevice-dev` is unnecessary. I will remove it when more dependencies are classified as unnecessary.
I just put almost everything which I installed when I started my trivial to build this project. Because the apt deps are so fundamental that it will spend a lot of time to test whether a dependency is necessary. So I just left them here.

Reply:
The two unnecessary ones that I can see are libplist-dev and libimobiledevice-dev. Everything else is necessary from what I can see.

Reply:
I removed